server/desktop-api.js

```diff
@@ -54,10 +53,109 @@ function serveTelemetryAPI (app) {
     })
   })
 
-  var basicAuth = auth(secret.credentials)
-  var fileServer = express.static(TELEMETRY_PATH)
-  var indexServer = serveIndex(TELEMETRY_PATH)
-  app.use('/desktop/telemetry', [basicAuth, indexServer, fileServer])
+  app.use('/desktop/telemetry/', [
+    auth(secret.credentials),
+    serveTelemetryDashboard,
+    express.static(TELEMETRY_PATH),
+    express.static(path.join(__dirname, '../static/telemetry-dashboard'))
```

Comment:
Can the static handlers go before `serveTelemetryDashboard `? Not a big deal, but it's a common express convention.